static/_redirects

```diff
@@ -537,7 +537,7 @@
 /id/docs/setup/minikube/     /id/docs/tasks/tools/ 302
 /docs/setup/learning-environment/     /docs/tasks/tools/  302!
 /id/docs/setup/learning-environment/     /id/docs/tasks/tools/  302!
-/zh/docs/setup/learning-environment/     /zh/docs/tasks/tools/  302!
+/zh-cn/docs/setup/learning-environment/     /zh-cn/docs/tasks/tools/  302!
```

Comment:
Maybe it's `/zh/docs/setup/learning-environment/` needs to be redirected instead of `/zh-cn/docs/setup/learning-environment/`.